src/cli/init.rs

```diff
@@ -62,6 +64,14 @@ const PYROJECT_TEMPLATE: &str = r#"
 name = "{{ name }}"
 channels = [{%- if channels %}"{{ channels|join("\", \"") }}"{%- endif %}]
 platforms = ["{{ platforms|join("\", \"") }}"]
+{%- for env, features in environments|items %}
+{%- if loop.first %}
+
+[tool.pixi.environments]
+default = {features = [], solve-group = "default"}
```

Comment:
Without the line `default = {features = [], solve-group = "default"}` `pixi install` does not work, and yields `missing python interpreter from environment`. I am not sure this is expected?

In any case, my understanding is this is the right syntax to solve all environments (including the default one) as part of the same solve-group; is that correct?

Reply:
I would say using the `"default"` solve-group without defining it in the "default" env would make sense to me. What do you think @pavelzw @baszalmstra?

But this error would be unexpected to me, maybe @baszalmstra know's what is going on? As I would have expected an `"default" solve group doesn't exist` error.

Reply:
Solve groups are just names you can define anywhere. The error is weird to me too, Ill check when I have the time.

Reply:
I meant to say that we could implicitly add the `solve-group = "default"` to the default env so users don't have to define it separately. WDYT?

Reply:
But “default” does not have a special meaning or anything? Other environments do not by default have the “default” solvegroup. What do you hope to achieve?

Reply:
I think it makes sense to have a "default" everything. 

- Default feature
- Default environment
- Default solve group

Reply:
Added Default SolveGroup in in https://github.com/prefix-dev/pixi/pull/1077/commits/96385f2eaf45090a6c78dc86684bc019bf6dc252

This will only work if the Default Environment is not redefined

Reply:
Also, I could not reproduce the issue with `missing python interpreter from environment`. so seems to be all good there.

Reply:
wouldn't this imply that every environment is in the same solve group if not specifying anything? Which was one thing we originally wanted to avoid?

Reply:
Currently, the only environment in the 'default' solve-group is the default environment.